pkg/client/client.go

```diff
@@ -17,9 +17,61 @@
 
 package client
 
+import (
+	"errors"
+	"github.com/dubbogo/dubbo-go-proxy/pkg/client/dubbo"
+	"github.com/dubbogo/dubbo-go-proxy/pkg/client/httpclient"
+	"github.com/dubbogo/dubbo-go-proxy/pkg/model"
+	_ "github.com/gin-gonic/gin"
+	"sync"
+)
+
 type Client interface {
 	Init() error
 	Close() error
-
 	Call(req *Request) (resp Response, err error)
 }
+
+type ClientPool struct {
```

Comment:
`ClientPool` is private or public scope?

Reply:
I think `ClientPool` should in private scope,  when use it, just run Call.